packages/core/nuxt-module/src/modules/server-url.ts

```diff
@@ -0,0 +1,15 @@
+export default function VueStorefrontServerUrl(): void {
+  const hasMiddlewareUrl = this.options?.publicRuntimeConfig?.middlewareUrl;
+  const hasSSRUrl = this.options?.publicRuntimeConfig?.ssrMiddlewareUrl;
+
+  if (!hasMiddlewareUrl || !hasSSRUrl) {
```

Comment:
If `middlewareUrl` is set and `ssrMiddlewareUrl` is not, it will still match criteria and go inside of `if`. It makes to pass `ssrMiddlewareUrl` also, which is not used anywhere and not documented.